packages/ai/package.json

```diff
@@ -63,6 +63,7 @@
     "@ai-sdk/provider-utils": "2.1.10",
     "@ai-sdk/react": "1.1.18",
     "@ai-sdk/ui-utils": "1.1.16",
+    "@modelcontextprotocol/sdk": "^1.6.0",
```

Comment:
Hm this is a major challenge because their dependencies match our, which can lead to dependency version conflicts and nasty bugs: https://www.npmjs.com/package/@modelcontextprotocol/sdk?activeTab=dependencies

Reply:
Will think about options how to address this. One is to move mcp tools to a separate pkg, another is to use peer dependencies.

Reply:
Looking into the solution you proposed most recently around calling web socket API directly!